src/components/views/create/NameInput.tsx

```diff
@@ -1,25 +1,62 @@
-import React from 'react';
-import { H5 } from '@giveth/ui-design-system';
-import { Regular_Input } from '@/components/styled-components/Input';
-import { InputContainer, Label } from './Create.sc';
+import React, { useState } from 'react';
+import { H5, neutralColors } from '@giveth/ui-design-system';
 import styled from 'styled-components';
+import {
+	InputContainer,
+	InputErrorMessage,
+	InputWithError,
+	Label,
+} from './Create.sc';
+import { FlexCenter } from '@/components/styled-components/Grid';
+import { ECreateErrFields } from '@/components/views/create/CreateIndex';
 
-const NameInput = (props: any) => {
-	const { setValue } = props;
+const NameInput = (props: {
+	value: string;
+	setValue: (e: string) => void;
+	error: string;
+}) => {
+	const { setValue, value, error } = props;
+	const [characterLength, setCharacterLength] = useState(0);
+	const maxLength = 55;
 	return (
 		<>
-			<H5>Name of your Project</H5>
+			<H5 id={ECreateErrFields.NAME}>Name of your Project</H5>
```

Comment:
I can't get why you add an id to it.

Reply:
on publish, we use that ID to scroll to the error